specification/recoveryservicesbackup/resource-manager/Microsoft.RecoveryServices/stable/2021-12-01/bms.json

```diff
@@ -9001,6 +9038,13 @@
         "retentionPolicy": {
           "$ref": "#/definitions/RetentionPolicy",
           "description": "Retention policy with the details on backup copy retention ranges."
+        },
+        "tieringPolicy": {
```

Comment:
>tieringPolicy [](http://example.com/codeflow?start=9&length=13)

New properties cannot be added in the context of an existing API version. This is considered a breaking change. Add the property to a new API version.

Reply:
As mentioned [above](https://github.com/Azure/azure-rest-api-specs/pull/17517#issuecomment-1022700466), 2021-12-01 is a new API version and these changes were missed in an earlier PR for this version.